src/main/scala/no/ndla/articleapi/service/converters/contentbrowser/VideoConverterModule.scala

```diff
@@ -24,6 +24,7 @@ trait VideoConverterModule {
       val (embedVideoMeta, errors) = HtmlTagGenerator.buildEmbedContent(Map(
         "resource" -> "brightcove",
         "id" -> s"${content.id.toString}",
+        "caption" -> content.get("link_text"),
```

Comment:
Dette er den eneste endringen som må til for å legge til captions på embed tagger av typen video.
Denne linjen gjør at det blir med en attributt "data-caption", med captionen som verdi, på embed taggen.